GoogleForCreators/web-stories-wp
# Dashboard: update storybook mock stories data

## Summary
Follows up earlier PR that added my stories views to storybook. Only use shapes and background colors for dummy story data to avoid local host image issues

## Testing Instructions
- Now if you go to Dashboard/Components/myStories/Content you should see shapes and colors. Since we don't have the utils necessary to render stories deeper in components you still won't see the colors/shapes in places like StoryGridView. But the myStories should show them for Default, All Data Fetched, All Data Fetched As List.  Note that it will not show on Stories View because this is just the stories component inside Content, does not have providers. 

<img width="767" alt="Screen Shot 2020-05-21 at 4 29 35 PM" src="https://user-images.githubusercontent.com/10720454/82616008-c74c3180-9b80-11ea-98e4-f8ec5263f01a.png">


---



Fixes #
